_resources/port1.0/group/crossgcc-1.0.tcl

```diff
@@ -261,6 +263,19 @@ proc crossgcc.setup {target version} {
         # fatal error: error in backend: ran out of registers during register allocation
         compiler.blacklist  {clang >= 421 < 422}
 
+        # Section taken from gcc11
```

Comment:
My suggestion is to add a comment to `gcc11` and `gcc-devel` with a hint for anyone modifying that port to also keep `crossgcc` in sync.

Yet another option (we can leave that one for later) would be to create a `gcc` PortGroup and let both `gcc` and `crossgcc` ports automatically include the exact same code for compiler selection, checksums, etc.

Reply:
I'll remove that comment, maybe @kencu can add that note into `gcc11` & `gcc-devel` as that really would be useful to keep `crossgcc` in sync.

Reply:
Oh, no, there's no need to remove that comment. I just wanted to suggest to add an additional one elsewhere. This comment here is useful.

Reply:
Ah, ok I've put it back just noticed this before heading out to work. I do agree a comment should be within `gcc11` and `gcc-devel` to keep this in sync.